client/main.js

```diff
@@ -124,3 +124,5 @@ Array.from( document.getElementsByClassName( 'wcc-root' ) ).forEach( ( container
 		container
 	);
 } );
+
+window.wcsAppStores = createdStores;
```

Comment:
Seems like it would be better to introduce an interface to get the store instead allowing direct access to them.
something like
```js
window.wcsGetAppStore( storeKey ) {
    return createdStores[ storeKey ];
}
```

Reply:
Would object references here be a concern? I am guessing not since it should be the same store globally. We shouldn't need to clone it every time under the same `window` object.

Reply:
@harriswong, I'm not totally clear on your question. Are you worried about wcsGetAppStore getting defined multiple times?

Reply:
I was thinking `createdStores ` is assigned to `window.wcsGetAppStore`. So anytime a component modify `createdStores`, the `window.wcsGetAppStore` updates. If component-x modifies `createdStores`, then component-y's `window.wcsGetAppStore` will get updated unexpectedly. But i think that's fine since the purpose of this function is to get the store based on a key.